agenta-backend/agenta_backend/models/db_models.py

```diff
@@ -1,303 +1,396 @@
+
+
+class TestSetDB(Base):
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    name = Column(String)
+    app_id = Column(Integer, ForeignKey("app_db.id"))
+    app = relationship("AppDB")
+    csvdata = Column(JSON)  # List of dictionaries
+    user_id = Column(Integer, ForeignKey("users.id"))
+    user = relationship("UserDB")
+    created_at = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+    updated_at = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+
+    __tablename__ = "testsets"
+
+
+class EvaluatorConfigDB(Base):
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    evaluation_id = Column(Integer, ForeignKey("evaluations.id"))
+    evaluation = relationship("EvaluationDB", back_populates="evaluator_configs")
+    evaluation_scenario_id = Column(Integer, ForeignKey("evaluation_scenarios.id"))
+    evaluation_scenario = relationship(
+        "EvaluationScenarioDB", back_populates="evaluator_configs"
+    )
+    app_id = Column(Integer, ForeignKey("app_db.id"))
+    app = relationship("AppDB")
+    user_id = Column(Integer, ForeignKey("users.id"))
+    user = relationship("UserDB")
+    name = Column(String)
+    evaluator_key = Column(String)
+    settings_values = Column(JSON, default=dict)
+    created_at = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+    updated_at = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+
+    __tablename__ = "evaluators_configs"
+
+
+class HumanEvaluationDB(Base):
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    app_id = Column(Integer, ForeignKey("app_db.id"))
+    app = relationship("AppDB")
+    user_id = Column(Integer, ForeignKey("users.id"))
+    user = relationship("UserDB")
+    status = Column(String)
+    evaluation_type = Column(String)
+    variant_id = Column(Integer, ForeignKey("app_variants.id"))
+    variant = relationship("AppVariantDB")
+    variant_revision_id = Column(Integer, ForeignKey("app_variant_revisions.id"))
+    variant_revision = relationship("AppVariantRevisionsDB")
+    testset_id = Column(Integer, ForeignKey("testsets.id"))
+    testset = relationship("TestSetDB")
+    created_at = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+    updated_at = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+
+    __tablename__ = "human_evaluations"
+
+
+class HumanEvaluationScenarioDB(Base):
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    user_id = Column(Integer, ForeignKey("users.id"))
+    user = relationship("UserDB")
+    evaluation_id = Column(Integer, ForeignKey("human_evaluations.id"))
+    evaluation = relationship("HumanEvaluationDB")
+    inputs = relationship("HumanEvaluationScenarioInputsDB", backref="scenario")
```

Comment:
Why do we have this in a different table? Why not use a json? Since these are the inputs that are related to one scenario. So it's a very small json.

Reply:
One scenario would especially in the future many inputs / outputs.
Also inputs outputs tend to be large data.
I think it's better to have them in a separate tables